contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -281,6 +288,19 @@ UninstallExtendedHooks(void)
  * 			Hook Functions
  *****************************************/
 
+static void
+pltsql_GetNewObjectId(VariableCache variableCache)
+{
+	if (!babelfish_dump_restore)
+		return;
+
+	if (ShmemVariableCache->nextOid >= babelfish_dump_restore_min_oid)
+		return;
+
+	ShmemVariableCache->nextOid = babelfish_dump_restore_min_oid;
+	ShmemVariableCache->oidCount = 0;
```

Comment:
what is the implication of setting oid count to 0

Reply:
That is for `XLogPutNextOid`.